### PR TITLE
[KB-270] Updated PCA sources

### PR DESCRIPTION
**Pull request by Ivikhostrup**
I removed the old sources we had regarding PCA. The justification for using these were weak as they were either unpublished papers or published in journals with bad reputation.

These new references fixes that.

### CODE REVIEW

File: report_thesis/src/sections/related_work.tex [hunk 1]
```diff
@@ -49,9 +49,17 @@ \subsection{Preprocessing and Feature Engineering}
 \subsection{Dimensionality Reduction Techniques}
 Dimensionality reduction techniques such as \gls{pca} play a crucial role in managing the high-dimensional nature of \gls{libs} data.
 
-\citet{rezaei_dimensionality_reduction} explored various machine learning techniques combined with \gls{pca} for dimensionality reduction. Their results demonstrated that \gls{svr} with \gls{pca} performed the best for different elements, highlighting the effectiveness of dimensionality reduction techniques in improving model performance by managing the complexities of high-dimensional data.
+\cite{pca_review_paper} conducted a comprehensive review of \gls{pca} applied in a \gls{libs} context. 
+Their review highlighted several studies that succesfully deployed \gls{pca}.
+For example, \cite{moncayo_pca} used \gls{pca} to analyze megapixel elemental maps composed of more than 1 million LIBS spectra.
```
Comment (chhoumann) [inline report_thesis/src/sections/related_work.tex:54]:
```suggestion
For example, \cite{moncayo_pca} used \gls{pca} to analyze megapixel elemental maps composed of more than 1 million \gls{libs} spectra. 
```

File: report_thesis/src/sections/related_work.tex [hunk 2]
```diff
@@ -49,9 +49,17 @@ \subsection{Preprocessing and Feature Engineering}
...
 Dimensionality reduction techniques such as \gls{pca} play a crucial role in managing the high-dimensional nature of \gls{libs} data.
 
-\citet{rezaei_dimensionality_reduction} explored various machine learning techniques combined with \gls{pca} for dimensionality reduction. Their results demonstrated that \gls{svr} with \gls{pca} performed the best for different elements, highlighting the effectiveness of dimensionality reduction techniques in improving model performance by managing the complexities of high-dimensional data.
+\cite{pca_review_paper} conducted a comprehensive review of \gls{pca} applied in a \gls{libs} context. 
+Their review highlighted several studies that succesfully deployed \gls{pca}.
+For example, \cite{moncayo_pca} used \gls{pca} to analyze megapixel elemental maps composed of more than 1 million LIBS spectra. 
+\gls{pca} successfully separated the contributions of various minerals, including those present in low amounts, highlighting the power of \gls{pca} in handling highly diverse samples.
+Another example is \citet{porizka_pca} used \gls{pca} to filter outliers and classify samples according to their matrix.
```
Comment (chhoumann) [inline report_thesis/src/sections/related_work.tex:56]: Which matrix

File: report_thesis/src/sections/related_work.tex [hunk 3]
```diff
@@ -49,9 +49,17 @@ \subsection{Preprocessing and Feature Engineering}
...
+\cite{pca_review_paper} conducted a comprehensive review of \gls{pca} applied in a \gls{libs} context. 
+Their review highlighted several studies that succesfully deployed \gls{pca}.
+For example, \cite{moncayo_pca} used \gls{pca} to analyze megapixel elemental maps composed of more than 1 million LIBS spectra. 
+\gls{pca} successfully separated the contributions of various minerals, including those present in low amounts, highlighting the power of \gls{pca} in handling highly diverse samples.
+Another example is \citet{porizka_pca} used \gls{pca} to filter outliers and classify samples according to their matrix. 
+This was followed by a univariate calibration of copper (\ce{Cu}) in soil samples, which resulted in a reduced bias. 
+Rather than mitigating the matrix effect, the use of PCA avoided it altogether.
```
Comment (chhoumann) [inline report_thesis/src/sections/related_work.tex:58]: They claim that it avoided...?

Comment (Ivikhostrup) [reply report_thesis/src/sections/related_work.tex:58]: I guess I could make this a stronger point. They basically said that they completely mitigated the matrix effects. I will rewrite.

File: report_thesis/src/sections/related_work.tex [hunk 4]
```diff
@@ -49,9 +49,18 @@ \subsection{Preprocessing and Feature Engineering}
 \subsection{Dimensionality Reduction Techniques}
 Dimensionality reduction techniques such as \gls{pca} play a crucial role in managing the high-dimensional nature of \gls{libs} data.
 
-\citet{rezaei_dimensionality_reduction} explored various machine learning techniques combined with \gls{pca} for dimensionality reduction. Their results demonstrated that \gls{svr} with \gls{pca} performed the best for different elements, highlighting the effectiveness of dimensionality reduction techniques in improving model performance by managing the complexities of high-dimensional data.
+\cite{pca_review_paper} conducted a comprehensive review of \gls{pca} applied in a \gls{libs} context. 
+Their review highlighted several studies that succesfully deployed \gls{pca}.
+For example, \cite{moncayo_pca} used \gls{pca} to analyze megapixel elemental maps composed of more than 1 million \gls{libs} spectra. 
+\gls{pca} successfully separated the contributions of various minerals, including those present in low amounts, highlighting the power of \gls{pca} in handling highly diverse samples.
```
Comment (Pattrigue) [inline report_thesis/src/sections/related_work.tex:55]:
```suggestion
\citet{pca_review_paper} conducted a comprehensive review of \gls{pca} applied within the context of \gls{libs}.
This review highlighted numerous studies that successfully utilized \gls{pca}.
For instance, \citet{moncayo_pca} used \gls{pca} to analyze megapixel elemental maps composed of over one million \gls{libs} spectra.
The \gls{pca} approach effectively separated the contributions of various minerals, including those present in low concentrations, demonstrating its robustness in handling highly diverse samples.
```

File: report_thesis/src/sections/related_work.tex [hunk 5]
```diff
@@ -49,9 +49,18 @@ \subsection{Preprocessing and Feature Engineering}
...
-This approach is particularly relevant to our work due to the focus on advanced statistical methods and machine learning algorithms for handling complex, high-dimensional \gls{libs} data, aligning with our objectives of improving accuracy and robustness in predicting major oxide compositions.
\ No newline at end of file
+Another example is \citet{porizka_pca} used \gls{pca} to filter outliers and classify samples according to their matrix composition, such as \ce{Al}, \ce{Ca}, \ce{Na}, and \ce{Si}. 
+This classification was followed by a univariate calibration of copper (\ce{Cu}) in soil samples, which resulted in a reduced bias. 
+\gls{pca} was employed to discriminate individual rocks based on their overall elemental composition and they showed that \gls{pca} effectively addressed matrix effects that can significantly impact the accuracy of analytical results.
+These studies among others highlighted by \citet{pca_review_paper} demonstrate the effectiveness of \gls{pca} as a preprocessing technique in \gls{libs} analysis.
```
Comment (Pattrigue) [inline report_thesis/src/sections/related_work.tex:60]:
```suggestion
In another example, \citet{porizka_pca} employed \gls{pca} to filter outliers and classify samples based on their matrix composition, including elements such as \ce{Al}, \ce{Ca}, \ce{Na}, and \ce{Si}. 
This classification was followed by a univariate calibration of copper (\ce{Cu}) in soil samples, resulting in reduced bias.
Additionally, \gls{pca} was used to discriminate individual rocks based on their overall elemental composition, effectively addressing matrix effects that can significantly impact the accuracy of analytical results.
These studies, among others highlighted by \cite{pca_review_paper}, underscore the effectiveness of \gls{pca} as a preprocessing technique in \gls{libs} analysis.
```

File: report_thesis/src/sections/related_work.tex [hunk 6]
```diff
@@ -49,9 +49,18 @@ \subsection{Preprocessing and Feature Engineering}
...
+
+\citet{sirven_pca_ann_plsr} investigated the influence of matrix effects on the performance of quantitative analysis of chromium (\ce{Cr}) in soil samples using \gls{libs}. 
+\gls{pca} was used to classify spectra from two different soils and to detect outliers. 
+\gls{pca} was successful in separating spectra from agricultural soil and kaolinite in the plane of the first two components. 
+Additionally, \gls{pca} was employed to identify and remove outliers from the data set, enhancing the accuracy of subsequent analyses using \gls{ann}s and \gls{plsr}.
+This study demonstrated the utility of \gls{pca} in managing matrix effects and improving the accuracy of quantitative \gls{libs} analysis.
```
Comment (Pattrigue) [inline report_thesis/src/sections/related_work.tex:66]:
```suggestion
\citet{sirven_pca_ann_plsr} investigated the influence of matrix effects on the performance of quantitative analysis of chromium (\ce{Cr}) in soil samples using \gls{libs}.
\gls{pca} was used to classify spectra from two different soils and to detect outliers.
It successfully separated spectra from agricultural soil and kaolinite in the plane of the first two components.
Furthermore, \gls{pca} was used to identify and remove outliers from the dataset, enhancing the accuracy of subsequent analyses using \gls{ann}s and \gls{plsr}.
This study demonstrated the utility of \gls{pca} in managing matrix effects and improving the accuracy of quantitative \gls{libs} analysis.
```

**Review by Pattrigue — APPROVED** [review]

**Review by chhoumann — APPROVED** [review]

**Pull request merged**